File: app/src/main/java/com/hamza/Wazaker/WorkManager/NotificationWorker.java
package com.hamza.Wazaker.WorkManager;


import android.content.Context;

import androidx.annotation.NonNull;
import androidx.work.Worker;
import androidx.work.WorkerParameters;

import com.hamza.Wazaker.Utills.Notifications;

public class NotificationWorker extends Worker {

    public NotificationWorker(@NonNull Context context, @NonNull WorkerParameters workerParams) {
        super(context, workerParams);
    }

    @NonNull
    @Override
    public Result doWork() {
        Notifications.addNotification(getApplicationContext());
        return Result.success();
    }
}

File: app/src/main/java/com/hamza/Wazaker/Worker/NotificationWorker.kt
package com.hamza.Wazaker.Worker

import android.content.Context
import androidx.work.Worker
import androidx.work.WorkerParameters
import com.hamza.Wazaker.Utills.Notifications

class NotificationWorker(context: Context, workerParams: WorkerParameters) :
    Worker(context, workerParams) {

    override fun doWork(): Result {
        Notifications.addNotification(applicationContext)
        return Result.success()
    }
}